data/json/items/tools.json

```
@@ -1968,6 +1968,60 @@
       "not_ready_msg": "The eggs are not done yet."
     }
   },
+  {
+    "id": "milk_cream_rising",
+    "type": "GENERIC",
+    "category": "food",
+    "name": "rising cream jar",
+    "description": "This jar contains raw milk separating into cream.  You can seal up the jar once the process is completed.",
+    "weight": "52 g",
```

Way too light. Here's the weight of a 3L jar + 12 units of raw milk:

```suggestion
    "weight": "3461 g",
```